src/extensions/DC/getDCOptions.js

```
@@ -62,10 +62,16 @@ wp.domReady(() => {
 
-const fetchUsers = async () => {
+const fetchUsers = async type => {
 	const { getUsers } = resolveSelect(coreStore);
-	const users = await getUsers();
-	return users ? users.map(({ id, name }) => ({ id, name })) : null;
+	let users = await getUsers();
+	if (type === 'customers') {
+		users = users?.filter(user => user?.roles?.includes('customer'));
+	}
+
+	return users && users.length
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)